src/main/resources/assets/js/test/authServiceTest.js

```diff
@@ -1,74 +1,70 @@
-define(['angular', 'angularMocks', 'application/services/tdpInvestAuthService'], function (angular) {
+define(['angular', 'angularMocks', 'application/auth/services/tdpInvestAuthService'], function (angular) {
 
     describe('tdpInvestAuthService', function () {
-        beforeEach(angular.mock.module('tdpInvestModule'));
+        beforeEach(angular.mock.module('tdpInvestAuthModule'));
 
         var $httpBackend;
-        var $service;
+        var service;
 
         beforeEach(inject(function (_tdpInvestAuthService_, _$httpBackend_) {
-            $service = _tdpInvestAuthService_;
+            service = _tdpInvestAuthService_;
             $httpBackend = _$httpBackend_;
+
+            spyOn(service, 'setCredentials');
+            spyOn(service, 'clearCredentials');
         }));
 
-        describe('When login', function () {
+        describe('When logging in', function () {
```

Comment:
describe(when logging in with valid credentials) { it(should succeed)} <- proper BDD style tests